docs/engine_questions.md

```diff
@@ -12,6 +12,13 @@ entity moving through the engine. Tokens are also not spawned and consumed for e
 * Activated elements
 * Taken sequence flows
 
+## How does the StreamPlatform and Engine interaction look like?
+
+Open the model and start the token flow to see the interaction between StreamProcessor (maintained by ZDP) and the RecordProcessor (Engine, maintained by the ZPA).
+
+[![engine-bpmn](https://user-images.githubusercontent.com/2758593/195591824-233ca579-96bc-4ee1-b6a3-5449929d75a9.png)](https://modeler.cloud.ultrawombat.com/diagrams/32c22fc0-709a-48d7-afe5-83a0c40a83fd--engine-bpmn?v=1076,651,1)
```

Comment:
I've created a milestone for the workflow at this moment, and made the change I suggested during the talk in a next milestone.

🔧 Please update the image to the latest state.

🔧 If you choose to ignore the above suggestion, please let me know so I follow up on it. IMO it makes sense to take it along now while we're changing this anyways.

Reply:
Thanks for that @korthout :+1: I also added my missing part regarding the acceptance call to the model. I updated the uploaded picture as well.